File: 27-RemoveElement/27-RemoveElement.cpp
// Last updated: 3/18/2026, 3:02:51 PM
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
         int c=0;
        for(int i=0; i<nums.size(); i++){
           if(nums[i] != val){
            swap(nums[i],nums[c]);
            c++;
            }
        }
        return c;
    }
};

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 3/18/2026, 3:02:55 PM
class Solution {
    public int removeElement(int[] nums, int val) {
        
        int k=0;

        for(int i=0; i<nums.length; i++){

            if(nums[i] != val){

                nums[k]=nums[i];
                k++;
            }
        }
        return k;
    }
}